vtexdocs/devportal
# fix(documentation-page): page margin in mobile screen

#### What is the purpose of this pull request?

Fix the margin on mobile screens (documentation page)

#### What problem is this solving?

Between the first two breakpoints (360-640), the margin was wrong, and the document content size wasn't following the screen size. The margin until `639px` will be `18px` and the content will expand to both sides.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-52--elated-hoover-5c29bf.netlify.app/docs/api-guides/clients) and check if the behavior explained above works.

#### Screenshots or example usage
| before | after |
| -------- | ------ |
|![image](https://user-images.githubusercontent.com/42784961/176454276-c6429928-5958-421c-9749-6e10ad74d2cd.png)|![image](https://user-images.githubusercontent.com/42784961/176454465-86f0a883-30a1-4c14-8cf2-f61e9e6270ad.png)|

#### Types of changes
